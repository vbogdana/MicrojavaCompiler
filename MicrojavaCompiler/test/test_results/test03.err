ERROR 05:58:00,895 - Semantics error at row 10 - expression 'a' must be a variable
ERROR 05:58:00,895 - Semantics error at row 11 - expression 'a' must be a variable
ERROR 05:58:00,895 - Semantics error at row 11 - incompatible types
ERROR 05:58:00,897 - Semantics error at row 13 - incompatible types
ERROR 05:58:00,897 - Semantics error at row 13 - expression must be a boolean value
ERROR 05:58:00,897 - Semantics error at row 14 - incompatible types
ERROR 05:58:00,899 - Semantics error at row 17 - 'break' statement can only be used inside a while loop
ERROR 05:58:00,899 - Parsing UNSUCCESSFUL!
