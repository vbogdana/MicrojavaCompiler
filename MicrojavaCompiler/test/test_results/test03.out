INFO  05:58:00,863 - Compiling source file: B:\STUDIJE\CETVRTA GODINA\sedmi semestar\PP\projekat\workspace\MicrojavaCompiler\test\test_files\test03.mj
INFO  05:58:00,891 - PARSING the program 'test03' at row 3
INFO  05:58:00,891 - Constant 'a' declaration at row 5
INFO  05:58:00,891 - Global variable 'x' declaration at row 6
INFO  05:58:00,893 - Global variable 'y' declaration at row 6
INFO  05:58:00,893 - Global variable 'z' declaration at row 7
INFO  05:58:00,893 - Function 'main' definition at row 9
INFO  05:58:00,893 - Use of Con a: int, 3, 0  at row 10
INFO  05:58:00,895 - Use of Con a: int, 3, 0  at row 11
INFO  05:58:00,895 - Use of Var x: int, 0, 0  at row 12
INFO  05:58:00,895 - Use of Var y: int, 1, 0  at row 12
INFO  05:58:00,895 - Use of Var z: char, 2, 0  at row 12
INFO  05:58:00,895 - Use of Var z: char, 2, 0  at row 13
INFO  05:58:00,897 - Use of Con a: int, 3, 0  at row 13
INFO  05:58:00,897 - Use of Var z: char, 2, 0  at row 14
INFO  05:58:00,897 - Use of Var y: int, 1, 0  at row 14
INFO  05:58:00,897 - Use of Var x: int, 0, 0  at row 14
INFO  05:58:00,897 - Use of Var x: int, 0, 0  at row 15
INFO  05:58:00,897 - Use of Var x: int, 0, 0  at row 15
INFO  05:58:00,899 - =======================SYNTAX ANALYSIS=======================
